src/pages/reporting/ExportAs.tsx

```diff
@@ -30,20 +31,42 @@ export const ExportAs = () => {
   };
 
   const getReportPdf = async () => {
-    fileSaverService.saveFile(
-      await dashboardService.getDashboardInfoPdf({
-        ...range,
-        'project-uuid': projectUuid,
-      }),
+    const exportRequestGuids = {
+      backupSize: mapPropertiesObjectListToStringOfGuids(
+        exportRequest.backupSize,
+      ),
+      transferSize: mapPropertiesObjectListToStringOfGuids(
+        exportRequest.transferSize,
+      ),
+      value: '',
+    };
+    await fileSaverService.saveFile(
+      await dashboardService.getDashboardInfoPdf(
+        {
+          ...range,
+        },
+        exportRequestGuids,
+      ),
     );
   };
 
   const getReportHtml = async () => {
-    fileSaverService.saveFile(
-      await dashboardService.getDashboardInfoHtml({
-        ...range,
-        'project-uuid': projectUuid,
-      }),
+    const exportRequestGuids = {
+      backupSize: mapPropertiesObjectListToStringOfGuids(
+        exportRequest.backupSize,
+      ),
+      transferSize: mapPropertiesObjectListToStringOfGuids(
+        exportRequest.transferSize,
+      ),
+      value: '',
+    };
+    await fileSaverService.saveFile(
+      await dashboardService.getDashboardInfoHtml(
```

Comment:
same.

Put `dashboardService.getDashboardInfoHtml` into constant, then provide it into `saveFile`